index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Cristian-Teddy Tribute Page</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+
+    <div id="main">
+      <div id="title">Dr. Norman Borlaug</div>
+
+      <div id="img-div">
+        <img id="image" src="image/norman-borlaug.jpg">
```

Comment:
- [ ] Image tags should always have an `<alt>` attribute containing a text that appears in place of the image in the event the image is unable to load. This text helps screen-reading tools and allows search engines to better crawl and rank your website.

You can refer to this [Hubspot article](https://blog.hubspot.com/marketing/image-alt-text)

Reply:
You may try putting it this way: `Please use <alt> attribute for all images in your code. This text helps screen-reading tools and allows search engines to better crawl and rank your website.`